### PR TITLE
Starlit Escape Update

### PR DESCRIPTION
**Pull request by Fraxinus88**
Ready to Run Update!

### CODE REVIEW

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/thestarlitescape/The Starlit Escape.sh [hunk 1]
```diff
@@ -1,20 +1,13 @@
 #!/bin/bash
-
-XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
-
 if [ -d "/opt/system/Tools/PortMaster/" ]; then
   controlfolder="/opt/system/Tools/PortMaster"
 elif [ -d "/opt/tools/PortMaster/" ]; then
   controlfolder="/opt/tools/PortMaster"
-elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
-  controlfolder="$XDG_DATA_HOME/PortMaster"
```
[the remark this answers is not in the document]
Comment (Fraxinus88) [reply ports/thestarlitescape/The Starlit Escape.sh:10]: Sorry this is from the old launch.sh

**Review by kloptops — APPROVED** [review]

File: runtimes/mono-6.12.0.122-aarch64.squashfs.part.004
[the remark this answers is not in the document]
Comment (Fraxinus88) [reply runtimes/mono-6.12.0.122-aarch64.squashfs.part.004:1]: No idea what the .py did :(

**Review by Cebion — CHANGES_REQUESTED** [review]
image

You're deleting, mono and updating the sonic3air port.json?

**Comment by Fraxinus88** [comment]
Let me just rebase this whole fork

**Pull request closed without merging**